src/app/components/authorized/add-new-subreddit/add-new-subreddit.component.ts

```diff
@@ -15,17 +17,16 @@ export class AddNewSubredditComponent {
         description: '',
     };
 
-    constructor(private _gatewayService: GatewayService) {}
+    errorMessage?: Error;
```

Comment:
errorMessage sounds like it is a string rather than an object.

Reply:
changed name as requested